graphics/zbar/Portfile

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+PortGroup               active_variants 1.1
+
+github.setup            mchehab zbar 0.23.90
+revision                0
+
+checksums               rmd160  ce5fb3bb7376ae0208cdb83baf706d447a11dfe5 \
+                        sha256  9581b5ab467f7b45ef0d7219e773c8dd0de07eb916f32f6828cd48b96ce50344 \
+                        size    1005439
+
+categories              graphics
+maintainers             @knapoc openmaintainer
+
+description             ZBar is an open source software suite for reading bar codes from \
+                        various sources
+long_description        ${description}, such as video streams, image files \
+                        and raw intensity sensors. It supports EAN-13/UPC-A, UPC-E, EAN-8, \
+                        Code 128, Code 93, Code 39, Codabar, Interleaved 2 of 5, QR Code and SQ Code.
+
+platforms               darwin
+license                 LGPL-2.1+
+
+depends_build           port:autoconf \
+                        port:automake \
+                        port:gettext \
+                        port:libtool \
+                        port:pkgconfig \
+                        port:xmlto
+depends_lib             port:ImageMagick \
+                        port:libjpeg-turbo
+
+use_autoreconf          yes
+autoreconf.args         -fvi
+configure.args          --disable-video \
+                        --disable-nls \
+                        --disable-silent-rules \
+                        --with-jpeg \
+                        --without-dbus
+
+# 'make test' doesn't actually do anything
+test.run                no
+
+variant python39 description {Include Python 3.9 binding} {
+                        depends_lib-append port:python39
+                        configure.env-append PYTHON=${prefix}/bin/python3.9
```

Comment:
please align this in a more conventional way, in other words `depends_lib-append` should just be indented with four spaces. Same applies for the other variants.